doc/topics/tutorials/cloud_controller.rst

```diff
@@ -260,7 +259,8 @@ one can not be found on the image using the default arguments to ``seed.apply``.
     installed will GREATLY speed up virtual machine deployment.
 
 You can also deploy an image on a particular minion by directly calling the
-`virt` execution module with an absolute image path. This can be quite handy for testing:
+`virt` execution module with an absolute image path. This can be quite handy for
```

Comment:
I know this isn't your original writing and since you're changing this line anyway, can you add another set of backticks around `virt` here? A trick of RST syntax is that it needs two backticks to format text `as code`, whereas markdown can use single or double backticks. Otherwise that virt reference will only show up as regular text and the backticks are ignored when the docs are rendered.